src/pybind/mgr/dashboard_v2/run-backend-api-tests.sh

```diff
@@ -77,6 +77,11 @@ export COVERAGE_FILE=.coverage.mgr.dashboard
 MGR=2 RGW=1 ../src/vstart.sh -n -d
 sleep 10
 
+# Create an admin user and set the credentials so that the dashboard can use them
+./bin/radosgw-admin user create --uid=admin --display-name=admin --system --access-key=admin --secret=admin
+./bin/ceph dashboard set-rgw-api-secret-key admin
+./bin/ceph dashboard set-rgw-api-access-key admin
```

Comment:
this should be done in our teuthology test instead.

Reply:
This will also be done for Teuthology, but has to be done in that file as well to ensure that the tests can be executed locally too.